Userland/Libraries/LibCpp/Parser.cpp

```diff
@@ -1237,6 +1237,16 @@ NonnullRefPtr<Type> Parser::parse_type(ASTNode& parent)
         type = ptr;
     }
 
+    if (!eof() && (peek().type() == Token::Type::And || peek().type() == Token::Type::AndAnd)) {
+        type->set_end(position());
+        auto ref = consume();
+        auto ptr = create_ast_node<Reference>(parent, type->start(), ref.end(), ref.type() == Token::Type::And ? Reference::Kind::Lvalue : Reference::Kind::Rvalue);
```

Comment:
nitpick: `ptr`=>`ref` (and `ref`=>`ref_token` or something :sweat_smile: : )

Reply:
Sure!